sdk/search/azure-search/src/main/java/com/azure/search/models/NamedEntityRecognitionSkillLanguage.java

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Defines values for NamedEntityRecognitionSkillLanguage.
+ */
+public enum NamedEntityRecognitionSkillLanguage {
```

Comment:
Should be string constants instead of an enum, but this is the only associated with the deprecated skill so maybe doesn't matter that much.

Reply:
@careyjmac I'm starting to think the scrubbing @arv100kri did might have been on the Track 1 Swagger only. We should diff those two and see what's changed between them.

Reply:
Either way, not blocking for this PR.